src/core/funcbloc.h

```diff
@@ -468,12 +470,11 @@ class CFunctionBlock : public forte::core::CFBContainer {
     void sendOutputEvent(TEventID paEO, CEventChainExecutionThread * const paECET){
       FORTE_TRACE("OutputEvent: Function Block sending event: %d (maxid: %d)\n", paEO, getFBInterfaceSpec().mNumEOs - 1);
 
-      #ifdef FORTE_TRACE_CTF
-        traceOutputEvent(paEO);
-      #endif
-
       if(paEO < getFBInterfaceSpec().mNumEOs) {
         writeOutputData(paEO);
+#ifdef FORTE_TRACE_CTF
```

Comment:
tracing the output event after write Outputdata is not changing the output of your event trace as you are tracing the values of the FB ports and not the connections. However moving that down changes the order of the generated traces for other users. Therefore I would kindly asking to move the output event tracing back where it was.

Reply:
I reverted it back